### ISSUE TITLE
[New Image] Replace the `potentialworkshop` image by a `potentialfit` image

### ISSUE THREAD

**Issue by mbruns91** [pyiron/docker-stacks#162]
The potentialworkshop image is based on the potentials workshop from 2021. Shall we maybe replace it ba an image providing developments from `pyiron_potentialfit`?

**Comment by jan-janssen** [comment]
Yes, this is the intention of the pull request https://github.com/pyiron/docker-stacks/pull/138 - still at the current stage the `pyiron_potentialfit` repository is not yet used in production. Furthermore, we might want to move parts of `pyiron_potentialfit` to the `atomistics` package to simplify the integration with `pyiron_workflow` in future.
